Platform.PCL/Realm.PCL/Schema/RealmSchemaPCL.cs

```diff
@@ -38,7 +38,14 @@ public class RealmSchema : IReadOnlyCollection<ObjectSchema>
         /// Gets the number of known classes in the schema.
         /// </summary>
         /// <value>The number of known classes.</value>
-        public int Count { get; }
```

Comment:
I was originally thinking of code calling Count independent of construction? You could have code which was in a PCL which received a `RealmSchema` object from elsewhere then called Count on it. In practice, I guess your simplification is worth it.